### PR TITLE
Function names rename for R/combine_analysis.R, R/combine_files.R, R/create_lineage_lookup.R, R/assign_job_queue.R

### PR DESCRIPTION
**Pull request by Seyi007**
# Description
Due to the last conversation on the Zoom meeting, I decided to separate the function names rename from the error handling PR I opened in #76. This makes it easier for the continuous topic-based merging currently being done by @the-mayer.

## What kind of change(s) are included?

- [ ] Feature (adds or updates new capabilities)
- [ ] Bug fix (fixes an issue).
- [x] Enhancement (adds functionality).
- [ ] Breaking change (these changes would cause existing functionality to not work as expected).

# Checklist

Please ensure that all boxes are checked before indicating that this pull request is ready for review.

- [x] I have read and followed the [CONTRIBUTING.md](CONTRIBUTING.md) guidelines.
- [x] I have searched for existing content to ensure this is not a duplicate.
- [x] I have performed a self-review of these additions (including spelling, grammar, and related).
- [x] I have added comments to my code to help provide understanding.
- [ ] I have added a test which covers the code changes found within this PR.
- [x] I have deleted all non-relevant text in this pull request template.
- [x] **Reviewer assignment**: @the-mayer @jananiravi

### CODE REVIEW

**Review by jananiravi — COMMENTED** [review]
@Seyi007 (@the-mayer) does this carry a combination of function renames, styler, and tryCatch error handling commits? Should we separate them out in the future into separate PRs, even if it's easier to go ahead with the combined one this time?

**Comment by the-mayer** [comment]
@jananiravi / @Seyi007 , I believe I have removed all code not relevant to the rename PR. This combined with a rebase to pull in additional PRs and I think we are ready to merge.

**Review by the-mayer — COMMENTED** [review]
@jananiravi / @Seyi007 , I believe I have removed all code not relevant to the rename PR. This combined with a rebase to pull in additional PRs and I think we are ready to merge.

**Comment by Seyi007** [comment]
Yes as @the-mayer said, this pr is solely for renaming functions. The other pr #76 is for error handling @jananiravi. This would make it easy for David to merge them on the basis of a specific issue.

File: R/assign_job_queue.R [hunk 1]
```diff
@@ -113,53 +128,58 @@ get_proc_medians <- function(dir_job_results) {
...
+#' writeProcessRuntime2YML(
 #'     "/data/scratch/janani/molevolvr_out/",
 #'     "/data/scratch/janani/molevolvr_out/log_tbl.yml"
 #' )
 #' }
 #' @export
-write_proc_medians_yml <- function(dir_job_results, filepath = NULL) {
-    if (is.null(filepath)) {
-        filepath <- file.path(common_root, "molevol_scripts", "log_data", "job_proc_weights.yml")
-    }
```
Comment (falquaddoomi) [inline R/assign_job_queue.R:157]: This check appears to be important; why was it removed?

**Review by falquaddoomi — COMMENTED** [review]
Generally, it looks good, and the new names are IMO much more readable. I left a few comments.

File: R/assign_job_queue.R (hunk 1 above)
Comment (the-mayer) [reply R/assign_job_queue.R:157]: Resolved with [df602df](https://github.com/JRaviLab/MolEvolvR/pull/95/commits/df602dfd63cbab0d84dbcc8229e3da9c7646b9d5)

File: R/acc2lin.R [hunk 2]
```diff
@@ -144,56 +150,64 @@ efetchIPG <- function(accnums, out_path, plan = "sequential", ...) {
...
+#' @description Takes the resulting file
+#'              of an efetch run on the ipg database and
 #'
 #' @param accessions Character vector of protein accessions
-#' @param ipg_file Filepath to the file containing results of an efetch run on the
-#' ipg database. The protein accession in 'accessions' should be contained in this
+#' @param ipg_file Filepath to the file
+#'                 containing results of an efetch run on the
+#' ipg database. The protein accession in
+#'               'accessions' should be contained in this
```
[the remark this answers is not in the document]
Comment (the-mayer) [reply R/acc2lin.R]: resolved with https://github.com/JRaviLab/MolEvolvR/pull/95/commits/1a0b66358eac637736a18868ae27e4049aa22628

**Pull request merged**